src/routes/(app)/order/[id]/+page.server.ts

```diff
@@ -109,7 +111,9 @@ export async function load({ params, depends, locals }) {
 			)
 		}),
 		overwriteCreditCardSvgColor: runtimeConfig.overwriteCreditCardSvgColor,
-		hideCreditCardQrCode: runtimeConfig.hideCreditCardQrCode
+		hideCreditCardQrCode: runtimeConfig.hideCreditCardQrCode,
+		labels,
+		adminPrefix: adminPrefix()
```

Comment:
This exposes the admin prefix to every user that makes an order.

The admin prefix is supposed to be a secret, it's the whole reason for why it exists.

By the way you can check the data sent by the bebop to the user by refreshing the page and doing Ctrl+U